src/Compilers/CSharp/Portable/Lowering/Instrumentation/DebugInfoInjector_SequencePoints.cs

```diff
@@ -63,7 +59,7 @@ private static TextSpan CreateSpanForConstructorInitializer(ConstructorDeclarati
             if (constructorSyntax.Modifiers.Any(SyntaxKind.StaticKeyword))
             {
                 // [SomeAttribute] static MyCtorName(...) [|{|] ... }
-                var start = constructorSyntax.Body.OpenBraceToken.SpanStart;
+                var start = constructorSyntax.Body!.OpenBraceToken.SpanStart;
```

Comment:
How do we know this can never be null?

Reply:
Constructors must have a body in syntactically correct tree.

Reply:
This could use `[MemberNotNullWhenNotNull]` attribute if it existed.